client/app/components/Input/__tests__/InputTextarea.spec.jsx

```diff
@@ -154,11 +155,13 @@ describe('InputTextarea', () => {
         { title: 'Unordered List', expectedArgs: ['insertUnorderedList'] },
       ];
 
-      buttons.forEach(({ title, expectedArgs }) => {
-        const button = screen.getByTitle(title);
-        userEvent.click(button);
-        expect(pell.exec).toHaveBeenCalledWith(...expectedArgs);
-      });
+      await Promise.all(
```

Comment:
Can we use `waitFor` instead?

Reply:
What's the rationale for `waitFor` vs `Promise.all`? I'm not sure I follow! My larger gut feeling is to break out of doing loops here entirely and write out all the Button conditions explicitly – just to avoid any hack-y-ness 🤔

Reply:
Could you do something like this instead with [waitFor](https://testing-library.com/docs/dom-testing-library/api-async/#waitfor)?

```
buttons.forEach(({ title, expectedArgs }) => {
  const button = screen.getByTitle(title);
  userEvent.click(button);
  
  await waitFor(() => expect(pell.exec).toHaveBeenCalledWith(...expectedArgs));
});
```

Reply:
Wahoo! Worked like a charm 🎉